recipes/libignition-sensors4/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set component_name = "sensors" %}
+{% set base_name = "libignition-" + component_name %}
+{% set version = "4.1.0" %}
+{% set major_version = version.split('.')[0] %}
+{% set name = base_name + major_version %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ignitionrobotics/ign-{{ component_name }}/archive/ignition-{{ component_name }}{{ major_version }}_{{ version }}.tar.gz
+    sha256: d3a3ae2d635323c0757a6d1bb17c5ccbd990ab491aa5136ad4934dbbeab6bebe
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - make  # [not win]
+    - cmake
+    - pkg-config
+  host:
+    - libignition-cmake2
+    - libprotobuf
+    - libignition-math6
+    - libignition-common3
+    - libignition-transport9
+    - libignition-rendering4
+    - libignition-msgs6
+    - libignition-plugin1
+    - libsdformat10
+  run:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/ignition/{{ component_name }}{{ major_version }}/ignition/{{ component_name }}.hh  # [not win]
+    - test -f ${PREFIX}/lib/libignition-{{ component_name }}{{ major_version }}.so  # [linux]
+    - test -f ${PREFIX}/lib/libignition-{{ component_name }}{{ major_version }}.dylib  # [osx]
+    - test -f ${PREFIX}/lib/cmake/ignition-{{ component_name }}{{ major_version }}/ignition-{{ component_name }}{{ major_version }}-config.cmake  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\ignition\\{{ component_name }}{{ major_version }}\\ignition\\{{ component_name }}.hh exit 1  # [win]
+    - if not exist $PREFIX$\\Library\\lib\\ignition-{{ component_name }}{{ major_version }}.lib (exit 0) exit 1  # [win]
```

Comment:
I think here is still a typo?
Instead of (exit 0) exit 1, it should just be (exit 1), right?

Reply:
Yes, fixed now, sorry for the noise.